react-native.html

```diff
@@ -0,0 +1,59 @@
+---
+layout: default
+title: Phoenix and Elixir | Next-gen tools for building modern, scalable web applications.
```

Comment:
@rozerosie this and the next line need an update

Reply:
should be fixed now; also updated the bottom of the page to match the phoenix/elixir page